rasa/core/agent.py

```diff
@@ -193,6 +194,29 @@ async def _schedule_model_pulling(
     )
 
 
+def create_anonymization_pipeline(
+    anonymization_rules,
+    event_broker_config,
+) -> Any:
+    anonymization_pipeline = plugin_manager().hook.create_anonymization_pipeline(
+        anonymization_rules=anonymization_rules,
+        event_broker_config=event_broker_config,
+    )
+
+    def _anonymizer(_, __, event_dict):
+        """Anonymizes log messages."""
+        text = event_dict["event"]
+        event_dict["event"] = anonymization_pipeline.log_run(text)
```

Comment:
Another thing to note, if the pipeline is None, then you need to return the event_dict as is.